underscore.mdx

```diff
@@ -5,133 +5,152 @@ description: Using underscore expressions to define features
 
 ---
 
-## Underscore expressions
+## Overview
 
-Underscore expressions can be used to define and
-resolve features from operations on other features.
+Underscore expressions are used to derive features from operations on other features. In feature definitions, `_`
+represents a reference to the containing feature class and is used to access other features in the same instance. In
+addition to arithmetic operations, underscore expressions can also be used to filter and aggregate data.
 
-Consider the feature class `RocketEngine` defined below:
+Underscore expressions are useful because Chalk statically analyzes and optimizes them, leading to better performance
```

Comment:
For this paragraph do we want to emphasize the performance benefits of underscores or the convenience of writing them compared to python resolvers? I don't really know or have an opinion because both are valuable to us and to customers. The convenience of using underscores really shows in the examples compared to the verbosity of actual resolver functions.